modules/cnf-topology-aware-lifecycle-operator-operator-and-platform-update.adoc

```diff
@@ -0,0 +1,124 @@
+// Module included in the following assemblies:
+// Epic CNF-2600 (CNF-2133) (4.10), Story TELCODOCS-285
+// * scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_content-type: PROCEDURE
+[id="talo-operator-and-platform-update_{context}"]
+= Performing a platform and an Operator update together
+
+.Prerequisites
+
+* Install the Topology Aware Lifecycle Operator.
+* Update ZTP to the latest version.
+* Provision one or more managed clusters with ZTP.
+* Log in as a user with `cluster-admin` privileges.
+* Create {rh-rhacm} policies in the Hub cluster.
+
+.Procedure
+
+. Create the `PolicyGenTemplate` CR for the updates by following the step described in Performing a platform update and Performing an Operator update sections.
+
+. Apply the `prep-work` for the platform and the Operator update.
+
+.. Save the content of the `ClusterGroupUpgrade` CR with the policies for platform update preparation work, catalog source updates, and target clusters to the `cgu-platform-operator-upgrade-prep.yml` file.
++
+[source,yaml]
+----
+apiVersion: ran.openshift.io/v1alpha1
+kind: ClusterGroupUpgrade
+metadata:
+  name: cgu-platform-operator-upgrade-prep
+  namespace: default
+spec:
+  managedPolicies:
+  - du-upgrade-platform-upgrade-prep
+  - du-upgrade-operator-catsrc-policy
+  clusters:
+  - spoke1
+  remediationStrategy:
+    maxConcurrency: 1
+  enable: true
+  actions:
+    afterCompletion:
+      deleteObjects: true
+----
+
+.. Apply the `cgu-platform-operator-upgrade-prep.yml` to the Hub cluster by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f cgu-platform-operator-upgrade-prep.yml
+----
+
+.. Monitor the process and wait for completion. Then ensure that the policy becomes compliant by running the following command:
++
+[source,terminal]
+----
+$ oc get policies --all-namespaces
+----
+
+. Pre-cache the images for the Operator update.
+.. Save the content of the Operator upgrade `ClusterGroupUpgrade` CR with the policies and the target clusters to the `cgu-platform-operator-upgrade.yml file`. This `ClusterGroupUpgrade` CR is configured to only perform the pre-caching job without the actual platform and Operator update.
++
+[source,yaml]
+----
+apiVersion: ran.openshift.io/v1alpha1
+kind: ClusterGroupUpgrade
+metadata:
+  name: cgu-du-upgrade
+  namespace: default
+spec:
+  managedPolicies:
+  - du-upgrade-platform-upgrade <1>
+  - du-upgrade-operator-catsrc-policy <2>
+  - common-subscriptions-policy <3>
+  preCaching: true
+  clusters:
+  - spoke1
+  remediationStrategy:
+    maxConcurrency: 1
+  enable: false
+  actions:
+    afterCompletion:
+      deleteObjects: true
+----
+<1> This is the platform update policy.
+<2> This is the policy to update the catalog source.
```

Comment:
Please change this to something like this: 

This is the policy containing the catalog source information for the operators to be upgraded. It is needed by the pre-caching feature to determine the operator images to be downloaded to spoke cluster.